pandunia-lekse.csv

```diff
@@ -627,12 +632,16 @@ dokumsanduke||||folder (directory)||carpeta|||||||||||||||
 dolare||||dollar||dolar|||||||||||||dolaro|dollari|dolar
 dolfine|||ell:δελφίνια (delphínia), eng:dolphin, spa:delfín, por:delfim, rus:дельфин (del’fin), ara:دلفين (dulfin), jpn:ドルフィン (dorufin), ben:ডলফিন (ḍalaphina), hin:डॉल्फ़िन (ḍŏlfin)|dolphin|dauphin|delfín|golfinho (delfim)|дельфин||海豚|イルカ (ドルフィン)|||||||yunus (delfin)|delfeno|delfiini|
 doma||||live in (reside in, inhabit)|vivre (habiter)|vivir (residir)|||||住む (居住する)||||||||loĝi|asua|mieszkać
+domdarsa||||tame (train)|apprivoiser (dresser, dompter, discipliner)|domar (amansar)||укрощать (приручить)||驯养|馴らす||||||||||
```

Comment:
This sounds like "home school" to me. I'm not sure is that a normal English term, but at least in Finnish it means that a kid has a teacher coming home instead of the kid going to school.

Reply:
that's exactly what it means in English. I did not notice that, but now that you've pointed it out it's pretty glaring.

then perhaps it would be better to have a root for tame (and have it mean obedient, as well), like **dosili** (Chinese **xuni** is also pretty good, but I think "docil" is known to slightly more people). to domesticate would then be **dosiljenisa**, to breed docile animals.

Reply:
I think that there are two sides in the word tame. One is about animals, which live at home with people. dom- is an important part of it. (See German "Haustier" for example!) The other side is about animals, which behave friendly towards people. There can be tame animals also in the wild because they are accustomed to people or they just don't care about people.

Reply:
I see what you mean. perhaps the best word for the first meaning is simply "domi". so "dosili" can describe the behavior of an animal that is calm around people, "dosiljenisi" can describe an animal that is docile from birth because it has been bred for the purpose of living with humans, and "domi" describes any animal (or person or object) that lives in the home.